exchange/websocket/manager.go

```diff
@@ -159,9 +160,14 @@ type ManagerSetup struct {
 // attempting a new connection. It also contains the subscriptions that are
 // associated with the specific connection.
 type connectionWrapper struct {
-	setup         *ConnectionSetup
-	subscriptions *subscription.Store
-	connection    Connection
+	setup           *ConnectionSetup
+	subscriptions   *subscription.Store
+	connectionsSubs []connectionSubscriptions
+}
+
+type connectionSubscriptions struct {
+	Connection    Connection
+	Subscriptions *subscription.Store
```

Comment:
Both can be unexported without issue. Resolve if it is somehow references externally